recipes/Google-speech-commands/README.md

```diff
@@ -14,6 +17,7 @@ python train.py hparams/xvect.yaml --data_folder=/path_to_/GSC --seed=1234  --nu
 |----------------- | ------------ |
 | xvector + augment v12 | 98.14% |
 | xvector + augment v35 | 97.43% |
+| xvector + augment + LEAF v35 | 96.79% |
```

Comment:
results are worst ?

Reply:
Yes. That's what I got in the first and only experiment